packages/backup-restore-operator/charts/templates/backup.yaml

```diff
@@ -0,0 +1,92 @@
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  name: backups.resources.cattle.io
+spec:
+  group: resources.cattle.io
+  names:
+    kind: Backup
+    plural: backups
+  scope: Namespaced
+  subresources:
+    status: {}
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            encryptionConfigName:
+              description: 'Name of secret containing the encryption config, must
+                be in the namespace of the chart: cattle-resources-system'
+              type: string
+            resourceSetName:
+              description: Name of resourceSet CR to use for backup, must be in the
+                same namespace
+              type: string
+            retentionCount:
+              type: integer
+            schedule:
+              description: Cron schedule for recurring backups
+              type: string
+            storageLocation:
+              nullable: true
```

Comment:
What happens if the storageLocation is `null`?

Reply:
The operator will try to use either the s3 backup location or pvc configured at the operator level. If not it errors. Describe this in more detail here https://rancher.atlassian.net/wiki/spaces/EN/pages/494501903/Backup+restore+design+for+docs+testing